data/856/680/81/85668081.geojson

```diff
@@ -853,7 +853,7 @@
     "wof:lang_x_spoken":[
         "spa"
     ],
-    "wof:lastmodified":1582358596,
+    "wof:lastmodified":1614808609,
     "wof:name":"Ciudad de Buenos Aires",
```

Comment:
Should this be `Buenos Aires` or `Autonomous City of Buenos Aires` or `City of Buenos Aires`? **name:eng_x_preferred** should also update.

Reply:
I agree, but the localization work is automated and based on mostly non-English names.. in this case, we probably should update these names, but I can't confirm/craft logic around this with the existing localizations I have. Maybe this can be a second pass?

Reply:
Sure